### DISCUSSION TITLE
关于标签的建议

### DISCUSSION THREAD

**Discussion by ChaoAiXue** [tamlok/vnote#1865] [Ideas]
Vnote2.0版本中, 标签之间是同一层级的关系

在3.0里面, 可不可以增加一个标签的分类哇, 便于索引的; 标签的好处是快速提取到关键笔记, 但是时间一长, 便签就 特别 特别 多T.T

下图是某记中实现的便签分类, (小声哔哔: 这个某记除了标签系统让人眼前一亮外,全是缺点...)

![image](https://user-images.githubusercontent.com/82295138/131805230-f0c06f94-a56f-4919-adfc-4ae1ca332e16.png)

**Comment by tamlok** [comment]
Should be supported.

**Reply by Dnnles** [reply]
这个标签是怎么用的
